packages/data-transport-layer/src/services/server/service.ts

```diff
@@ -48,9 +48,12 @@ const optionSettings = {
       return validators.isUrl(val) || validators.isJsonRpcProvider(val)
     },
   },
-  showUnconfirmedTransactions: {
-    validate: validators.isBoolean,
-  },
+  defaultSource: {
+    default: 'batched',
+    validate: (val: string) => {
+      return val === 'batched' || val === 'sequenced'
```

Comment:
will we have more sources than `batched` or `sequenced` in the future? assuming no since it seems like they directly correspond to `l1` and `l2` right?

Reply:
if we anticipate adding more sources 
```javascript
['batched', 'sequenced'].includes(val)
```

Reply:
They don't necessarily correspond to L1 and L2 per say, really the abstraction is batched or batched + pre-batched. The batched transactions are currently batched to Ethereum L1 but will be batched to an Eth2 data shard in the future.

Working on unifying the language between the different codebases.

> `['batched', 'sequenced'].includes(val)`

We only have 2 sources right now so I think its fine. I prefer writing JS using a subset of the language that is as C like as possible with Python sprinkled in, see https://github.com/handshake-org/hsd/blob/master/lib/blockchain/chain.js as an example